ols/src/llms/llm_loader.py

```diff
@@ -1,32 +1,19 @@
 """LLM backend libraries loader."""
 
-import inspect
 import logging
-import warnings
 from typing import Optional
 
-from genai import Client, Credentials
-from genai.extensions.langchain import LangChainInterface
-from genai.text.generation import TextGenerationParameters
-from ibm_watson_machine_learning.foundation_models import Model
-from ibm_watson_machine_learning.foundation_models.extensions.langchain import (
-    WatsonxLLM,
-)
-from ibm_watson_machine_learning.metanames import (
-    GenTextParamsMetaNames as GenParams,
-)
 from langchain.llms.base import LLM
-from langchain_openai import ChatOpenAI
 
-from ols import constants
 from ols.app.models.config import ProviderConfig
+from ols.src.llms.providers.registry import LLMProvidersRegistry
 from ols.utils import config
 
 logger = logging.getLogger(__name__)
 
 
 # workaround to disable UserWarning
-warnings.simplefilter("ignore", UserWarning)
+# warnings.simplefilter("ignore", UserWarning)
```

Comment:
I didn't find a difference when set and when commented, so I would be for removing.
Do you know what is the reason behind?

Reply:
I'd remove it, and also remove the `import warnings`